core/imodel-bridge/src/imodelJsWrapper/ConfigMapper.ts

```diff
@@ -0,0 +1,177 @@
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+const fs = require("fs");
+import { get } from "request-promise-native";
+
+export class ConfigMapper {
+
+  private _argumentsMap: Map<string, string | undefined> | undefined;
+  private _wrapperArgumentsMap: Map<string, string | undefined> | undefined;
+
+  public constructor() {
+
+  }
+
+  public  async initializeConfigVariables(): Promise<void> {
+
+    return new Promise((resolve, reject) => {
+
+      try {
+
+        this._argumentsMap = new Map<string, string> ();
+        this._wrapperArgumentsMap = new Map<string, string> ();
+
+        // Bridge variables
+        this._argumentsMap.set ("server-repository", process.env.IMODEL_NAME);
+        this._argumentsMap.set ("server-project-guid", process.env.CONTEXT_ID);
+        this._argumentsMap.set ("server-environment", process.env.SERVER_ENVIRONMENT);
+        if (process.env.DMS_TYPE != null) {
+          this._argumentsMap.set ("dms-library", process.env.DMS_LIBRARY);
+          if (undefined !== process.env.DMS_INPUT_FILE_URL)
+            this._argumentsMap.set ("dms-inputFileUrn", process.env.DMS_INPUT_FILE_URL);
+          this._argumentsMap.set ("dms-type", process.env.DMS_TYPE);
+          this._argumentsMap.set ("dms-documentGuid", process.env["imbridge--dms-documentGuid"]);
+          this._argumentsMap.set ("dms-datasource", process.env["imbridge--dms-datasource"]);
+          this._argumentsMap.set ("dms-accessToken", process.env["imbridge--dms-accessToken"]);
+          this._argumentsMap.set ("dms-workspaceDir", process.env["imbridge--dms-workspaceDir"]);
+        }
+        this._argumentsMap.set ("thirdParty-accessToken", process.env["imbridge--thirdParty-accessToken"]);
+        this._argumentsMap.set ("fwk-staging-dir", process.env.FWK_STAGINGDIR);
+        this._argumentsMap.set ("fwk-jobrun-guid", process.env.FWK_JOBRUN_GUID);
+        // Local Variables
+        this._wrapperArgumentsMap.set ("server-nodeOidcCallBackUrl", process.env.TOKEN_URI);
+        this._wrapperArgumentsMap.set ("server-nodeAuthguid", `Token ${process.env.AUTH_GUID}`);
+
+        this._wrapperArgumentsMap.set ("Bridge_Name", process.env.Bridge_Name);
+        this._wrapperArgumentsMap.set ("BridgeExePath", process.env.FWK_BRIDGEDIR);
+        this._wrapperArgumentsMap.set ("imodelId", process.env.IMODEL_ID);
+        this._wrapperArgumentsMap.set ("BridgeName", process.env.BRIDGE_NAME);
+        this._wrapperArgumentsMap.set ("InputFileName", process.env.INPUT_FILE);
+        this._wrapperArgumentsMap.set ("WorkingDir", process.env.FWK_WORKDIR);
+
+        // Set iModelJs config variable for region code
+        switch (process.env.SERVER_ENVIRONMENT) {
+          case "DEV":
+            process.env.imjs_buddi_resolve_url_using_region = "103";
```

Comment:
I believe there are standard enums defined in imodle.js for it. So need not duplicate this.